packages/mdc-select/mdc-select.scss

```diff
@@ -15,69 +15,60 @@
 //
 
 @import "@material/animation/functions";
+@import "@material/animation/variables";
 @import "@material/typography/mixins";
 @import "@material/theme/mixins";
+@import "@material/ripple/mixins";
 @import "@material/rtl/mixins";
 
 @mixin mdc-select-dd-arrow-svg-bg_($fill-hex-number: 000000, $opacity: .54) {
   background-image: url(data:image/svg+xml,%3Csvg%20width%3D%2210px%22%20height%3D%225px%22%20viewBox%3D%227%2010%2010%205%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%0A%20%20%20%20%3Cpolygon%20id%3D%22Shape%22%20stroke%3D%22none%22%20fill%3D%22%23#{$fill-hex-number}%22%20fill-rule%3D%22evenodd%22%20opacity%3D%22#{$opacity}%22%20points%3D%227%2010%2012%2015%2017%2010%22%3E%3C%2Fpolygon%3E%0A%3C%2Fsvg%3E);
 }
 
+$mdc-select-arrow-padding: 26px;
+$mdc-select-label-padding: 16px;
+$mdc-select-menu-transition: transform 180ms $mdc-animation-standard-curve-timing-function;
+
 // postcss-bem-linter: define select
 .mdc-select {
-  $dd-arrow-padding: 24px;
-
   @include mdc-typography(subheading2);
   @include mdc-theme-prop(color, text-primary-on-light);
-  @include mdc-rtl-reflexive-box(padding, right, $dd-arrow-padding);
   @include mdc-select-dd-arrow-svg-bg_;
 
   display: inline-flex;
+  position: relative;
   align-items: center;
   justify-content: flex-start;
-  max-width: calc(100% - #{$dd-arrow-padding});
-  height: 32px;
-  transition:
-    mdc-animation-exit-temporary(border-bottom-color, 150ms),
-    mdc-animation-exit-temporary(background-color, 150ms);
+  box-sizing: border-box;
+  height: 56px;
   border: none;
-  border-bottom: 1px solid rgba(black, .12);
-  border-radius: 0;
-  background-color: transparent;
+  border-radius: 4px 4px 0 0;
+  outline: none;
   background-repeat: no-repeat;
-  background-position: right center;
-  font-family: Roboto, sans-serif;
```

Comment:
FYI this was also removed in a PR that was just merged. Not sure if this will cause a merge conflict next time you merge from master.